### PR TITLE
lab01 to submit

### PR DESCRIPTION
**Pull request by santorsm**
Submitting django lab01

### CODE REVIEW

**Review by cdmichaelb — CHANGES_REQUESTED** [review]
Code Errors on attempting to access server after running.

> FieldError at /
> Cannot resolve keyword 'task_complete' into field. Choices are: completed_date, created_date, id, priority, priority_id, text

**Comment by cdmichaelb** [comment]
[d32b4ef](https://github.com/PdxCodeGuild/class_liger/pull/140/commits/d32b4efcb4e69427fda286eae2e4586f6151f1ce) corrects issue. Will re-review.

File: code/matt/django/lab01/django_to_do/to_do_app/templates/layout/coompleted.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{% extends 'layout/base.html' %} 
+
```
Comment (cdmichaelb) [inline code/matt/django/lab01/django_to_do/to_do_app/templates/layout/coompleted.html:2]: Typo in this file name, but the file doesn't seem to be used.

File: code/matt/django/lab01/django_to_do/to_do_app/views.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        "to_do_items": to_do_items,
+        "to_do_complete": to_do_complete,
+        "to_do_incomplete": to_do_incomplete,
+    }
+
+    return render(request, "layout/index.html", context)
+    # return render(request, "layout/index.html", context)
+
+
+def create(request):
```
Comment (cdmichaelb) [inline code/matt/django/lab01/django_to_do/to_do_app/views.py:30]: Errors if the user attempts to run a GET request on `create` (http://127.0.0.1:8000/create/)

Suggestion: Handle get requests either be returning an error, or returning to index.

**Review by cdmichaelb — APPROVED** [review]
Looks good, all features of the page function properly within the scope of the assignment.

**Comment by perennialAutodidact** [comment]
Nice work!

A few notes:
1. Watch out for erroneous imports. The most recent versions of VS Code will import things automatically when tab is pressed to auto-complete stuff. I had to remove an unnecessary import from one of the `models.py` files to get the server to start.
2. Model names are written in the singular because they represent the attributes for a single item from the table. (e.g. Priority vs Priorities)

**Pull request merged**